config/locales/en.yml

```diff
@@ -157,6 +157,12 @@ en:
           - "Yes"
           - "No"
           - "Not sure"
+      afford_rent_mortgage_bills:
```

Comment:
I'd suggest moving this up to line 120, so the questions remain in the same order as the Figma prototype (https://www.figma.com/file/tibZIxmeYW0Z3qEL4CEF72/Triage-tool---prototypes?node-id=953%3A2).

Reply:
Thanks! Have amended.